rust-toolchain

```diff
@@ -1,5 +1,5 @@
 [toolchain]
 channel = "stable"
 components = [ "rustc", "cargo", "clippy", "rustfmt", "rust-analyzer" ]
-targets = [ "wasm32-unknown-unknown" ]
+targets = [ "wasm32-unknown-unknown", "x86_64-unknown-linux-gnu"]
```

Comment:
Noob question, why is this needed?

Reply:
I saw that in dev/up the target was being added

This change isn't necessary, but would remove the need for that

Reply:
Ah, looks like it's for the validation service, guess we're building it for a docker container on `linux/amd64`